src/sync/commits.ts

```diff
@@ -26,14 +23,16 @@ export const getCommitTask = async (
 	logger: LoggerWithTarget,
 	_github: GitHubAPI,
 	gitHubClient: GitHubInstallationClient,
-	_jiraHost: string,
+	jiraHost: string,
 	repository: Repository,
 	cursor?: string | number,
-	perPage?: number): Promise<CommitData> => {
-	logger.info("Syncing commits: started");
-	const { edges, commits } = await fetchCommits(gitHubClient, repository, cursor, perPage);
+	perPage?: number): Promise<TaskPayload<CommitQueryNode, JiraCommitData>> => {
+
+	logger.debug("Syncing commits: started");
+	const timeCutoff = await numberFlag(NumberFlags.SYNC_MAIN_COMMIT_TIME_LIMIT, NaN, jiraHost);
+	const { edges, commits } = await fetchCommits(gitHubClient, repository, timeCutoff, cursor, perPage);
 	const jiraPayload = await transformCommit({ commits, repository });
-	logger.info("Syncing commits: finished");
+	logger.debug("Syncing commits: finished");
```

Comment:
Just thinking if info is actually the right level here... would be easy to understand if the sync is stuck during commits sync, wdyt? (And same above where you changed it for "started")